### PR TITLE
Fixes #8041: Typo in webapp logs

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8041

### CODE REVIEW

File: inventory-provisioning-web/src/main/scala/com/normation/inventory/provisioning/endpoint/FusionReportEndpoint.scala [hunk 1]
```diff
@@ -149,7 +149,7 @@ class FusionReportEndpoint(
                       save(certifiedReport)
                     } else {
                       // Signature is not valid, reject inventory
-                      val msg = s"RejectiInventory '${inventory}' for Node '${report.node.main.id.value}' because signature is not valid, you can update the inventory key by running the following command '/opt/rudder/bin/rudder-keys change-key ${report.node.main.id.value} <your new public key>'"
+                      val msg = s"RejectInventory '${inventory}' for Node '${report.node.main.id.value}' because signature is not valid, you can update the inventory key by running the following command '/opt/rudder/bin/rudder-keys change-key ${report.node.main.id.value} <your new public key>'"
```
Comment (VinceMacBuche) [inline inventory-provisioning-web/src/main/scala/com/normation/inventory/provisioning/endpoint/FusionReportEndpoint.scala]: Can you say instead "Rejecting inventory" please ? :)

**Comment by amousset** [comment]
PR updated

File: inventory-provisioning-web/src/main/scala/com/normation/inventory/provisioning/endpoint/FusionReportEndpoint.scala (hunk 1 above)
Comment (amousset) [inline inventory-provisioning-web/src/main/scala/com/normation/inventory/provisioning/endpoint/FusionReportEndpoint.scala]: Sure!

**Comment by VinceMacBuche** [comment]
Did you add the file ? :D no changes :(

**Comment by amousset** [comment]
PR updated

**Comment by amousset** [comment]
Should be better now!

**Comment by VinceMacBuche** [comment]
Thanks ;)

**Pull request merged**